apps/spruce/src/pages/task/taskTabs/TaskHistory/index.tsx

```diff
@@ -108,6 +123,8 @@ const Container = styled.div`
   display: flex;
   flex-direction: column;
   gap: ${size.s};
+
+  height: calc(100vh - 270px);
```

Comment:
nit: This works well but doesn't account for banners. Not sure if it's worth to add the calculation.

Reply:
you're right, the height is kind of wonky, i will check how to fix it